src/tests/ftest/cart/util/cart_logtest.py

```diff
@@ -238,11 +238,11 @@ def has_data(self):
         return self.__hwm != 0
 
     def __str__(self):
-        return "Total:{:,} HWM:{:,} {} allocations, {} frees".\
-            format(self.__val,
-                   self.__hwm,
-                   self.__acount,
-                   self.__fcount)
+        return 'Total:{:,} HWM:{:,} {} allocations,'.format(self.__val,
```

Comment:
Not worth re-pushing for this, but if you do, you might want a space here
```suggestion
        return 'Total:{:,} HWM:{:,} {} allocations, '.format(self.__val,
```